layouts/partials/settings.html

```diff
@@ -1,6 +1,18 @@
 {{- range $i, $def := (getJSON (printf "themes/tvpage/defaults/%s.json" .Type)).option -}}
```

Comment:
Check if you can use range in the simplest version (without  $i, $def :=) since these variables are not being used.